iina/AppDelegate+Scripting.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  AppDelegate+Scripting.swift
+//  iina
+//
+//  Created by Nate Weaver on 2020-03-02.
+//  Copyright © 2020 lhc. All rights reserved.
+//
+
+import AppKit
+
+extension AppDelegate {
+
+  func application(_ sender: NSApplication, delegateHandlesKey key: String) -> Bool {
+    return Set(["orderedPlayerWindows", "orderedPlayers"]).contains(key)
```

Comment:
For this few items, this could probably just be an explicit check–i.e. `key == "orderedPlayerWindows" || key == "orderedPlayerWindows"`. Unless you think there's a good reason to not do it this way?

Reply:
Probably just a holdover from code I had in another app with a *lot* more matches (to reduce noise).

Reply:
Actually, `orderedPlayerWindows` isn't even a thing currently, so I'll remove it.